infrastructure/cluster/flux-v2/reana/reana-release.yaml

```diff
@@ -1,84 +1,90 @@
-# apiVersion: helm.toolkit.fluxcd.io/v2beta1
-# kind: HelmRelease
-# metadata:
-#   name: reana
-#   namespace: reana
-#   annotations:
-#     flux.weave.works/automated: "false" 
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: reana
+  namespace: reana
+  annotations:
+    flux.weave.works/automated: "false" 
    
-# spec:
-#   releaseName: reana
-#   interval: 5m
-#   chart:
-#     spec:
-#       sourceRef:
-#         kind: HelmRepository
-#         name: reana
-#         namespace: reana
-#       chart: reana
-#       interval: 1m
-#       version: 0.9.1-alpha.3
+spec:
+  releaseName: reana
+  interval: 5m
+  chart:
+    spec:
+      sourceRef:
+        kind: HelmRepository
+        name: reana
+        namespace: reana
+      chart: reana
+      interval: 1m
+      version: 0.9.1-alpha.3
 
-#   values:
+  valuesFrom:
+    - kind: Secret
```

Comment:
Another possibility here would be to combine this into one secret and merge the values of the secret with this config.

Reply:
Reana expects the DB secret `reana-db-secrets` to be there, and the helm chart follows that logic. The IAM and Openstack share secrets are instead custom ones, and I would keep them separate for clarity.

Reply:
I think my previous comment was confusing, what I meant was to create a value file with the key and value mappings accordingly, rather than targeting the keys individually, but I guess that is just a style preference.